rpc/wrtc_client.go

```diff
@@ -180,8 +180,15 @@ func dialWebRTC(
 	exchangeCtx, exchangeCancel := context.WithCancel(signalCtx)
 	defer exchangeCancel()
 
+	// bool representing whether initial sdp exchange has occurred
+	haveInit := true
```

Comment:
I don't think this variable is doing anything useful right now

Reply:
`haveInit` is useful for returning WebRTC client errors when two init stages are received or an update comes in before an init. I agree, though, that it seems like the two of you have found that `sendErr` won't actually happen until an init stage has been received? So, it seems like we don't need to move it up here, and there's no need to check its value in `sendErr`?

Reply:
I think `haveInit` is useful yes -- sorry if I was being cryptic. IIUC the code in this PR initializes `haveInit` to true and nothing ever changes that.

Reply:
Ah true; if we _were_ to keep it here, it should probably start out as false.